tests/floris_interface_test.py

```diff
@@ -15,6 +15,22 @@ def test_read_yaml():
     assert isinstance(fi, FlorisInterface)
 
 
+def test_reinitialize():
```

Comment:
Let's test this on the FlowField class directly rather than indirectly through the FlorisInterface

Reply:
Will do!